ui-tests/src/test/java/org/openmrs/reference/page/PatientCaptureVitalsPage.java

```diff
@@ -138,6 +139,13 @@ public void confirm() {
             confirmButton.click();
         } catch (Exception e) {}
     }
+    
+    public void saveChanges() {
+        try {
+            WebElement saveChangesButton = findElement(SAVE_CHANGES_BUTTON);
+            saveChangesButton.click();
+        } catch (Exception e) {}
```

Comment:
Add a return function or a message in the catch block here.

Reply:
@sherrif10, I have read that void methods don't return a value, I will only add a return; Will that be better?

Reply:
Add may be a message or catch an exceptions something like **e,getMessage()**

Reply:
> @sherrif10, I have read that void methods don't return a value, I will only add a return; Will that be better?

and since its a void method, we could not use return, and only use e.getMessage()  @Irenyak1

Reply:
@sherrif10  should i add the message to all the methods in that class? There are many that were already existing but without a return function or message. Or should I only deal with only the methods I have introduced in that class?

Reply:
> There are many that were already existing but without a return function or message

Which methods could be these ones

Reply:
Why do we catch this exception? Shouldn't the test just fail?

Reply:
Hello @dkayiwa , i logged a ticket https://issues.openmrs.org/browse/RATEST-235  to handle all the return messages, Ideally I think its not practically good to handle such exceptions in this kind of approach, we can at least have a message indicating either whether the test fail or pass

Reply:
Why do you even have to catch it in the first place?

Reply:
i think to have a full description catch block.

Reply:
What happens if we do not have this try catch?

Reply:
Wow , @Irenyak1 , any response on the above comment

Reply:
Been off for few minutes and found this long conversation. Great idea @dkayiwa!  I think it would be a good idea for us to avoid these exceptions since they bring a certain amount of overhead with them.  So we could completely do away with them and also have that new ticket created by @sherrif10 just implement the removal of all these exceptions.

Reply:
So are you removing this one?

Reply:
Sure @dkayiwa for this particular ticket, let me remove the exception I introduced then the rest will be removed while working on the other ticket. How about that?

Reply:
That will be awesome, men/lady! 👍

Reply:
I have worked on it. Could you look at it again @dkayiwa .